schema.yaml

```diff
@@ -1,4 +1,61 @@
-application_api_version: v1beta1
+x-google-marketplace:
+  schemaVersion: v2
+
+  # Must match the version of the Application custom resource object.
+  applicationApiVersion: v1beta1
+
+  # The release version must match the release tag on the deployer
+  publishedVersion: '1.4.0'
+
+  publishedVersionMetadata:
+    releaseNote: >-
+      Update marketplace tools so app works with Kubernetes version >= 1.16.
+
+    # If "recommended" is "true", users using older releases are encouraged
+    # to update as soon as possible. This is useful if, for example, this
+    # release fixes a critical issue.
+    recommended: true
+
+  # This MUST be specified to indicate that the deployer supports managed
+  # updates. Note that this could be left out or kalmSupported set to false,
+  # in which case the deployer uses schema v2 but does not support updates.
+  managedUpdates:
+    kalmSupported: false
+
+  images:
+    postgres:
+      properties:
+        imageRepo:
+          type: REPO_WITH_REGISTRY
+        imageTag:
+          type: TAG
+    nginx:
+      properties:
+        imageRepo:
+          type: REPO_WITH_REGISTRY
+        imageTag:
+          type: TAG
+    init:
+      properties:
+        imageInitFull:
+          type: FULL
+        imageInitRegistry:
+          type: REGISTRY
+        imageInitRepo:
+          type: REPO_WITHOUT_REGISTRY
+        imageInitTag:
+          type: TAG
+
+  clusterConstraints:
+    resources:
+    - requests:
+        cpu: 1000m
```

Comment:
Ditto on if this is necessary

Reply:
See comment above. I can change this if there's a better value.